### PR TITLE
TsFile V4 for Table Model

### PR DESCRIPTION
**Pull request by jt2594838**

### CODE REVIEW

File: .github/workflows/unit-test.yml
Comment (HTHou) [inline .github/workflows/unit-test.yml]: This part is useless anymore and can be removed.

File: cpp/pom.xml
Comment (HTHou) [inline cpp/pom.xml]: Should we keep the version 1.0.1-SNAPSHOT? We can change it later after the PR merged.

File: .github/workflows/unit-test.yml
Comment (jt2594838) [reply .github/workflows/unit-test.yml]: Removed

File: cpp/pom.xml
Comment (jt2594838) [reply cpp/pom.xml:25]: This PR is just to trigger the CI, and I will not merge it into 'develop'.
Actually, I will merge it back to tsfile_v4.

Comment (jt2594838) [reply cpp/pom.xml]: How about we update it to 1.2.0-SNAPSHOT now?

**Review by JackieTien97 — APPROVED** [review]

**Review by HTHou — APPROVED** [review]

**Pull request merged**
